src/main/resources/assets/admin/common/js/rest/HeadRequest.ts

```diff
@@ -0,0 +1,29 @@
+import {Request} from './Request';
+import {BrowserHelper} from '../BrowserHelper';
+import {HttpMethod} from './HttpMethod';
+import {UriHelper} from '../util/UriHelper';
+
+export class HeadRequest
+    extends Request {
+
+    constructor() {
+        super(HttpMethod.HEAD);
+    }
+
+    protected prepareRequest(): void {
+        super.prepareRequest();
+
+        this.request.setRequestHeader('Accept', 'application/json');
```

Comment:
Why would HEAD request only accept `application/json` ?

Reply:
Well, because it is a copy from GetRequest which had that :) 
Agree that it is not necessary since head doesn't load body at all. 
It doesn't do any harm either since it is a hint to server what client is expecting, but not a requirement.